jekyll/_includes/global-nav.html

```diff
@@ -37,7 +37,7 @@
         {% include_cached mobile-sidebar.html site=site page=page %}
         <div class="global-nav--list global-nav--external-links lang-selector-wrapper">
           <button id="globe-lang-btn" class="global-nav--language-picker" data-analytics-action="click-outer-cta">
-            <span class="globe-icon">{% include utility-icons/globe.svg size="24" %}</span>
+            <img src='/docs/assets/img/compass/globe.svg' class="globe-icon"></img>
```

Comment:
If you put this into preview '/doc's get replaces with preview url and your image won't work. you will want to replace it with {{ site.baseurl }}
```suggestion
            <img src='{{ site.baseurl }}/assets/img/compass/globe.svg' class="globe-icon"></img>
```

Reply:
awesome, thanks for the tip on this! 🎉